client/layout/guided-tours/tours/simple-payments-email-tour/index.js

```diff
@@ -35,8 +35,8 @@ const handleTargetDisappear = () => {
 };
 
 // IE9+ polyfill for `Element.matches()` used in `DelegatingQuit`
-if ( ! Element.prototype.matches ) {
-	Element.prototype.matches = Element.prototype.msMatchesSelector;
+if ( ! window.Element.prototype.matches ) {
```

Comment:
Is this just extra clean up along the way, or related to this PR?

Reply:
This fixes the 
> org.eslint.no-undef - /home/circleci/wp-calypso/client/layout/guided-tours/tours/simple-payments-email-tour/index

lint error mentioned earlier.

Reply:
This lint fix is not directly related to the goal of the PR. It's a driveby fix.